src/router.ts

```diff
@@ -139,6 +141,72 @@ export default class Router {
       }
     });
 
+    router.get(
+      '/gas-refund/last-epoch-merkle-root/:network',
+      async (req, res) => {
+        try {
+          const network = Number(req.params.network);
+          const gasRefundApi = GasRefundApi.getInstance(network);
+          const epochInfo = EpochInfo.getInstance(network);
+
+          const currentEpochNum = await epochInfo.getCurrentEpoch();
+          const lastEpochNum = currentEpochNum - 1;
+          const gasRefundDataLastEpoch =
+            await gasRefundApi.gasRefundDataForEpoch(lastEpochNum);
+
+          return res.json(gasRefundDataLastEpoch);
+        } catch (e) {
+          logger.error(req.path, e);
+          res.status(403).send({
+            error: `GasRefundError: could not retrieve merkle root for last epoch`,
+          });
+        }
+      },
+    );
+
+    router.get(
+      '/gas-refund/all-merkle-data/:network/:address',
+      async (req, res) => {
+        const address = req.params.address;
+
+        try {
+          const network = Number(req.params.network);
+          const gasRefundApi = GasRefundApi.getInstance(network);
+          const gasRefundDataAddress =
+            await gasRefundApi.getAllGasRefundDataForAddress(address);
+
+          return res.json(gasRefundDataAddress);
+        } catch (e) {
+          logger.error(req.path, e);
+          res.status(403).send({
+            error: `GasRefundError: could not retrieve merkle data for ${address}`,
+          });
+        }
+      },
+    );
+
+    router.get('/gas-refund/describe/:network/:epoch', async (req, res) => {
+      const epoch = Number(req.params.epoch);
+      const network = Number(req.params.network);
+
+      try {
+        const gasRefundApi = GasRefundApi.getInstance(network);
+
+        const [root, leaves] = await Promise.all([
+          gasRefundApi.gasRefundDataForEpoch(epoch),
+          gasRefundApi.getAllEntriesForEpoch(epoch),
+        ]);
+        return res.json({
+          root,
+          leaves,
```

Comment:
This can end up a very big response...

Reply:
I know, this this is to help debug data. Can put a @todo or @fixme remove to not forget but I need it just now